src/rebar3_coveralls.erl

```diff
@@ -125,10 +125,12 @@ do_coveralls(ConvertAndSend, Get, GetLocal, MaybeSkip, Task) ->
   Report0 =
     #{service_job_id => ServiceJobId,
       service_name   => ServiceName},
+  Report1 = collect_git_info(Report0),
```

Comment:
Make this step conditional on service being GitHub, running a shell command if the output isn't needed seems like a unnecessary.